src/mame/psikyo/psikyo.cpp

```diff
@@ -1761,62 +1763,72 @@ Chips:  PS2001B
         PS3204
         PS3305
 
-4-U59      security (PIC16C57; not dumped)
+4-U59      security (PIC16C57)
 
 ***************************************************************************/
 
 ROM_START( tengai )
-	ROM_REGION( 0x100000, "maincpu", 0 )        /* Main CPU Code */
+	ROM_REGION( 0x100000, "maincpu", 0 )        // Main CPU Code
 	ROM_LOAD32_WORD_SWAP( "5-u40.bin", 0x000000, 0x080000, CRC(90088195) SHA1(8ec48d581ecd14b3dad36edc65d5a273324cf3c1) ) // 1&0
 	ROM_LOAD32_WORD_SWAP( "4-u41.bin", 0x000002, 0x080000, CRC(0d53196c) SHA1(454bb4695b13ce44ca5dac7c6d4142a8b9afa798) ) // 3&2
 
-	ROM_REGION( 0x020000, "audiocpu", 0 )       /* Sound CPU Code */
+	ROM_REGION( 0x020000, "audiocpu", 0 )       // Sound CPU Code
 	ROM_LOAD( "1-u63.bin", 0x00000, 0x20000, CRC(2025e387) SHA1(334b0eb3b416d46ccaadff3eee6f1abba63285fb) )
 
-	ROM_REGION( 0x001000, "mcu", 0 )       /* MCU */
-	ROM_LOAD( "4-u59.bin", 0x00000, 0x01000, NO_DUMP )
+	ROM_REGION( 0x001000, "mcu", 0 )       // MCU, not hooked up
+	/* PIC configuration:
+	     -User ID: 37EA
+	     -Watchdog Timer: ENABLE
+	     -Oscilator Mode: RC
+	*/
```

Comment:
PCB pictures show the PIC microcontroller with a 4.00 MHz crystal and a pair of ceramic disc capacitors connected across pins 26 and 27.  The oscillator mode is most likely XT (crystal/resonator), but could possibly be HS (high-speed crystal/resonator).  You’re only seeing RC in the dump because the configuration word was erased to deprotect the device.

Reply:
So, it's better to remove there comments?

Having the PCB, how can we determine the parameters?

Thanks!!!!

Reply:
It’s misleading to add a comment based on the value of an erased configuration word.

You can make an educated guess by looking at what‘s connected to pins 26 and 27:
* If it’s a resistor and capacitor on pin 27, it’s RC mode.
* If it’s a crystal or resonator with a higher frequency than 4 MHz, it’s HS mode.
* If it’s a crystal or resonator with a frequency between 100 kHz and 4 MHz, it’s probably XT mode, although HS mode will also work at the high end of that range.
* If it’s a crystal or resonator with a frequency less than 100 kHz, it’s probably LP mode.
* If it’s a logic level clock signal connected to pin 27 (e.g. derived from the main CPU clock), it could be LP, XT or HS mode.

You can clearly see on Tengai and Strikers 1945 PCBs that it uses a 4 MHz crystal with ceramic disc load capacitors and a 47 Ω series resistor.  It can’t possibly be RC mode or LP mode.  It’s probably XT mode, but HS mode would also work.